site/_guides/cert-manager.md

```diff
@@ -120,8 +118,9 @@ There are plenty of other ways to deploy cert-manager, but they are out of scope
 
 To keep things simple, we skip cert-manager's Helm installation, and use the supplied YAML manifests:
 
-```
-$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.8.0/cert-manager.yaml
+```sh
+$ kubectl create namespace cert-manager
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
```

Comment:
Does this command create the required namespace? It doesn't include a -n cert-manager flag, so it looks like the previous command isn't needed.

Reply:
Yeah, I suspect this is to allow the management of the `kubectl delete` flow.

Not deleting the namespace as part of the YAML means that you can `kubectl delete -f <file>` and you won't get stuck with the namespace not terminating because of the ValidatingWebhook config.